### PR TITLE
chore: Added Metadata in package.json

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
## Related Issue

Closes: #214 

### Describe the changes you've made

- Added the metadata for `package.json`

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/215"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,5 +26,22 @@
     "eslint-config-next": "12.3.0",
     "postcss": "^8.4.5",
     "tailwindcss": "^3.0.13"
+  },
+  "description": "webx dao website",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Pradumnasaraf/WebXDAO.github.io.git"
```
Comment (Panquesito7) [inline package.json:33]:
```suggestion
    "url": "git+https://github.com/WebXDAO/WebXDAO.github.io.git"
```

File: package.json [hunk 2]
```diff
@@ -25,5 +26,22 @@
...
+    "javascript",
+    "typescript",
+    "tailwindcss",
+    "nextjs",
+    "webxdao"
+  ],
+  "author": "WebXDAO",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Pradumnasaraf/WebXDAO.github.io/issues"
```
Comment (Panquesito7) [inline package.json:45]:
```suggestion
    "url": "https://github.com/WebXDAO/WebXDAO.github.io/issues"
```

File: package.json [hunk 3]
```diff
@@ -25,5 +26,22 @@
     "eslint-config-next": "12.3.0",
     "postcss": "^8.4.5",
     "tailwindcss": "^3.0.13"
+  },
+  "description": "webx dao website",
```
Comment (Panquesito7) [inline package.json:30]:
```suggestion
  "description": "WebXDAO website",
```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "webxdao.github.io",
```
Comment (Panquesito7) [inline package.json:2]: Hmm, we might want to change the name here. What do you think?

Comment (Pradumnasaraf) [reply package.json:2]: It goods repo name goes there

File: package.json (hunk 1 above)
Comment (Pradumnasaraf) [reply package.json]: Thank you, I am so silly.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀

**Review by krshkodes — APPROVED** [review]
Thank you!

**Pull request merged**